mvp-gate-party/src/main/kotlin/cz/tyckouni/mvpgate/party/persistence/sp/SpRepository.kt

```diff
@@ -1,5 +1,14 @@
 package cz.tyckouni.mvpgate.party.persistence.sp
 
 import cz.tyckouni.mvpgate.party.persistence.PartyRepository
+import java.util.*
 
-interface SpRepository : PartyRepository<Sp>
+interface SpRepository : PartyRepository<Sp> {
+
+    /**
+     * Find ServiceProvider by its name
+     *
+     * @param name SP name used for search
+     */
+    fun findByName(name: String): List<Sp>
```

Comment:
With JPA and hibernate it is more common to use `Set` instead of `List` since there is no order guaranteed wihtout sorting.

Reply:
That sounds good, changed.